src/NHibernate/Persister/PersisterFactory.cs

```diff
@@ -127,18 +115,12 @@ public static class PersisterFactory
 		}
 
 		public static ICollectionPersister Create(System.Type persisterClass, Mapping.Collection model,
-												  ICacheConcurrencyStrategy cache, ISessionFactoryImplementor factory, Configuration cfg)
+												  ICacheConcurrencyStrategy cache, ISessionFactoryImplementor factory)
 		{
 			ConstructorInfo pc;
-			var use4Parameters = false;
 			try
 			{
 				pc = persisterClass.GetConstructor(CollectionPersisterConstructorArgs);
-				if (pc == null)
-				{
-					use4Parameters = true;
-					pc = persisterClass.GetConstructor(CollectionPersisterConstructor2Args);
```

Comment:
Could this removal be a breaking change? (Is it possible for NHibernate users to supply custom persister to NHibernate?)